src/utils.ts

```diff
@@ -180,7 +180,28 @@ export function getDaysArray(start: Date, end: Date) {
  * 0 goes to yellow-50, 0.5 goes to yellow-100, etc. up to 10
  */
 export function floatToColor(float: number): string {
-  if (float === -1) return `rgba(255,255,255,1)`;
+  if (float === -1) return `rgba(234, 235, 234,1)`;
+  if (float === 0) return `rgba(239, 69, 56,1)`;
+  if (float === 0.5) return `rgba(241, 95, 84,1)`;
+  if (float === 1) return `rgba(246, 118, 108,1)`;
+  if (float === 1.5) return `rgba(245, 136, 128,1)`;
+  if (float === 2) return `rgba(247, 154, 148,1)`;
+  if (float === 2.5) return `rgba(249, 169, 163,1)`;
+  if (float === 3) return `rgba(249, 202, 198,1)`;
+  if (float === 3.5) return `rgba(250, 209, 206,1)`;
+  if (float === 4) return `rgba(252, 231, 229,1)`;
+  if (float === 4.5) return `rgba(251, 238, 237,1)`;
+  if (float === 5) return `rgba(255,255,255,1)`;
+  if (float === 5.5) return `rgba(240, 248, 240,1)`;
+  if (float === 6) return `rgba(222, 239, 223,1)`;
+  if (float === 6.5) return `rgba(203, 231, 204,1)`;
+  if (float === 7) return `rgba(181, 221, 183,1)`;
+  if (float === 7.5) return `rgba(160, 211, 162,1)`;
+  if (float === 8) return `rgba(138, 201, 141,1)`;
+  if (float === 8.5) return `rgba(112, 190, 115,1)`;
+  if (float === 9) return `rgba(94, 181, 97,1)`;
+  if (float === 9.5) return `rgba(76, 173, 80,1)`;
+  if (float === 10) return `rgba(64, 148, 68,1)`;
```

Comment:
Delete the code below it, also use a switch statement instead of giant if chain.